app/views/layouts/application.html.haml

```diff
@@ -27,7 +27,7 @@
         = yield
     = render :partial => "layouts/footer"
 
-    = render :partial => "layouts/scripts"
+    = render :partial => "layouts/scripts" unless Rails.env.test? || ENV['TRAVIS']
```

Comment:
Nice! :+1: I would only change it to `if Rails.env.production?`
- It reads easier.
- It also stops Piwik to run on developer machine.